app/controllers/api/lists_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::ListsController < ApiController
+  before_action :authenticated?
+  
+  def index
+    lists = List.all
+    render json: lists, each_serializer: ListSerializer
+  end
+  
+  def create 
+    list = List.new(list_params)
```

Comment:
same thing about the association as above, but with the user. Get the user id from the params hash and associate this list with the user that created it.